Jenkinsfile

```diff
@@ -18,7 +18,7 @@ pipeline {
     }
     stage('Test') {
       steps {
-        sh './test.sh'
+        sh 'summon ./test.sh'
```

Comment:
can this be run locally still?

Reply:
Yes, this follows the same pattern for integrations tests that we use for the service broker. The test script checks to see if the summon variable are present, and conditionally runs the integration tests if they are [1].

When this is run locally without summon it will simply skip the integration tests. This can also be run locally with summon connected to ConjurOps to run the integration tests on developer machines as well. 

[1] https://github.com/cyberark/cloudfoundry-conjur-buildpack/pull/36/files#diff-1b531c5e405e0bc95890d52cf2f2f5b5R14

Reply:
I've updated the CONTRIBUTING guide to include this note and add instructions for the integration tests:

https://github.com/cyberark/cloudfoundry-conjur-buildpack/pull/36/commits/85619be3714149e2c1266021c15b8d1224c4ffc4